catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -502,6 +502,11 @@
         <bundle>mvn:ddf.catalog.async/catalog-async-processingframework/${project.version}</bundle>
     </feature>
 
+    <feature name="catalog-harvest-api" version="${project.version}"
+        description="API for Harvesting components">
+        <bundle>mvn:ddf.catalog.harvest/catalog-harvest-api/${project.version}</bundle>
+    </feature>
```

Comment:
❓ This feature will not be started by default, right?

Reply:
Correct.